src/otx/algo/utils/xai_utils.py

```diff
@@ -287,7 +287,7 @@ def _convert_labels_from_hcls_format(
 
 def set_crop_padded_map_flag(explain_config: ExplainConfig, datamodule: OTXDataModule) -> ExplainConfig:
```

Comment:
Not related to this PR, but for a future reference `explain_config` should ideally be one of the next args to remove :)

Reply:
I will review this later. Thank you for the notice!